tests/e2e/bin/wp-setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#####
+# This script is designed to be running inside the WP container
+# it is basically a hacky way to insert arbitrary PHP code into wp-config without messing with bash escaping etc.
+# Also, it creates a debug log file
+
+touch wp-content/debug.log
+chown www-data:www-data wp-content/debug.log
+
+sed -i "/\/\* That's all, stop editing! Happy publishing. \*\//i\
```

Comment:
Editing this line probably doesn't happen often (last changed 22 months ago), but any harm in looking for the opening `<?php` line and adding these immediately following it?

Non-blocking.